postmark/django_backend.py

```diff
@@ -156,14 +156,14 @@ def _send(self, messages):
             if to_send is False:
                 # The message was missing recipients.
                 # Bail.
-                return False
+                return False, None
         else:
             pm_messages = list(map(self._build_message, messages))
             pm_messages = [m for m in pm_messages if m]
```

Comment:
If I send three messages, but one of the messages doesn't have a recipient, then at this stage the `pm_messages` list will have only two elements.

How is user-code supposed to figure out which of the three messages didn't get send? That is, after this point, there's no difference between:

```
connection.send_messages([wrong_message, correct_message1, correct_message2])
connection.send_messages([correct_message1, wrong_message, correct_message2])
connection.send_messages([correct_message1, correct_message2, wrong_message])
```

Would be great if the return value could then be

```
[None, message_id1, message_id2]
[message_id1, None, message_id2]
[message_id1, message_id2, None]
```

in these cases?

Reply:
Good call. Will track via https://github.com/themartorana/python-postmark/issues/108